shared/actions/fs/index.js

```diff
@@ -988,6 +989,35 @@ const onFSOverallSyncSyncStatusChanged = (state, action) =>
     progress: Constants.makeSyncingFoldersProgress(action.payload.params.status.prefetchProgress),
   })
 
+const notifyDiskSpaceStatus = (diskSpaceStatus: Types.DiskSpaceStatus) => {
+  switch (diskSpaceStatus) {
+    case 'error':
+      NotifyPopup('Sync Error', {
+        body: 'You are out of disk space. Some folders could not be synced.',
+        sound: true,
+      })
+      break
+    case 'warning':
+      NotifyPopup('Disk Space Low', {body: 'You have less than 1 GB of storage space left.'})
+      break
+    case 'ok':
+      break
+    default:
+      Flow.ifFlowComplainsAboutThisFunctionYouHaventHandledAllCasesInASwitch(diskSpaceStatus)
+  }
+}
+
+let prevOutOfSpace = false
```

Comment:
@jakob223 why is this a file-scoped variable instead of being in the state?

Reply:
Hmm, I'm not really sure I have a good reason. Feel free to move it to the state, or I can put up a PR for that in a bit if you want.

Reply:
No worries, I'll do it in my notification PR